devlib/instrument/gem5power.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, target, power_sites):
             self.power_sites = power_sites
         else:
             self.power_sites = [power_sites]
-        self.add_channel('sim_seconds', 'time')
+        self.add_channel('timestamp', 'time')
```

Comment:
Thanks for pointing that out. I've added a mapping for the site name that will allow for `sim_seconds` to be used when extracting the timestamp from the data which should fix that problem.